cve_bin_tool/cli.py

```diff
@@ -545,6 +545,20 @@ def main(argv=None):
             )
         )
 
+    if (
+        not args["input_file"]
+        and args["directory"]
+        and os.path.isfile(args["directory"])
+    ):
+        if (
+            args["directory"].endswith(".csv")
+            or args["directory"].endswith(".json")
+            or args["directory"].endswith(".vex")
+        ):
+            args["input_file"] = args["directory"]
+            args["directory"] = ""
+            LOGGER.warning("The scan will be performed with --input_file parameter")
```

Comment:
I'm not sure if users will know what this means, so here's a stab at rewording:

```suggestion
            LOGGER.warning("File specified is not a binary file and will be scanned as a bill of materials.")
            LOGGER.warning("Specify --input_file to avoid this warning in future scans.")
```

Reply:
thanks!